contracts/protocols/polygon/aave_v2/main.sol

```diff
@@ -53,6 +53,11 @@ contract Resolver is AaveHelpers {
         AaveAddressProvider addrProvider = AaveAddressProvider(getAaveAddressProvider());
         data = getList(AaveLendingPool(addrProvider.getLendingPool()));
     }
+
+    function getPrice(address[] memory tokens) public view returns (TokenPrice[] memory tokenPrices, uint256 ethPrice) {
```

Comment:
@KaymasJain Added this function to get priceFeed for testing purposes !!
Is it okay?

Reply:
Sure!